### ISSUE TITLE
CSV files have strange characters on OSX

### ISSUE THREAD

**Issue by nicolasdevops** [open-dict-data/ipa-dict#24]
Several languages csv files are missing from the download file at https://github.com/open-dict-data/ipa-dict/releases/tag/1.0
including Korean, Khmer and Icelandic. However I think it's feasible to get them from the txt files at https://github.com/open-dict-data/ipa-dict/tree/master/data

**Comment by dohliam** [comment]
@nicolasdevops Yes, the last release was quite some time ago and newer languages have not been included. I can generate a new release which should add all the current languages.

Btw, the title of this issue alludes to a different problem, but you don't mention any details. Which of the CSV files are you noticing strange characters in? Can you provide an example?
